<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="CRTV" ignored="0" name="Test1" tests="3" failures="2" timestamp="2023-10-26T00:44:09 IST" time="0.041" errors="0">
  <testcase name="validation" time="0.007" classname="selenum_4.TestNG_validation">
    <failure type="java.lang.AssertionError" message="Actual is not equal to exptecd">
      <![CDATA[java.lang.AssertionError: Actual is not equal to exptecd
at selenum_4.TestNG_validation.validation(TestNG_validation.java:19)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- validation -->
  <testcase name="validation2" time="0.002" classname="selenum_4.TestNG_validation"/>
  <testcase name="validation3" time="0.002" classname="selenum_4.TestNG_validation">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	second assertion expected [Yahoo] but found [google],
	3rd assertion expected [gogle] but found [google]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
second assertion expected [Yahoo] but found [google],
3rd assertion expected [gogle] but found [google]
at selenum_4.TestNG_validation.validation3(TestNG_validation.java:47)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- validation3 -->
</testsuite> <!-- Test1 -->
